### PR TITLE
Feature/content notification push

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: notifications-push@.service [hunk 1]
```diff
@@ -27,6 +27,8 @@ ExecStart=/bin/sh -c '\
       --env "QUEUE_HOST=kafka" \
       --env "API_BASE_URL=$API_BASE_URL" \
       --env "CONSUMER_BACKOFF=2" \
+      --env "NOTIFICATIONS_RESOURCE=content" \
+      --env "WHITELIST=^http://.*-transformer-(pr|iw)-uk-.*\.svc\.ft\.com(:\d{2,5})?/(content)/[\w-]+.*$" \
```
Comment (Nesze) [inline notifications-push@.service]: If you could add the mapper here also, it would be great.

Comment (izzyblues) [inline notifications-push@.service]: What's the mapper URL look like?

Comment (Nesze) [inline notifications-push@.service]: Just adding an OR to transformer should work

> ...-(transformer|mapper)-...

Comment (izzyblues) [inline notifications-push@.service]: 👍

**Pull request merged**